packages/venia-ui/upward.yml

```diff
@@ -12,7 +12,7 @@ veniaResponse:
         # handled by the top-level 'veniaProxy' object, which is a ProxyResolver
         # that passes the request through to the backing Magento server.
         - matches: request.url.pathname
-          pattern: '^/(graphql|rest|media)(/|$)'
+          pattern: '^/(magento_version|graphql|rest|media)(/|$)'
```

Comment:
@tjwiebell suggested that this may not be a good idea: https://magento.slack.com/archives/G8N21AQTV/p1608225831014200?thread_ts=1608224666.012800&cid=G8N21AQTV